diff --git a/alberta/src/Common/Makefile.am.include b/alberta/src/Common/Makefile.am.include
index 97ba728..f9562cb 100644
--- a/alberta/src/Common/Makefile.am.include
+++ b/alberta/src/Common/Makefile.am.include
@@ -2,6 +2,7 @@
#
# Makefile.am fragment shared by all builds
#
+pkgconfigdir=$(libdir)/pkgconfig

ASSEMBLE_SRC = \
../Common/M_M_M_M_assemble_fcts.c\
diff --git a/alberta_util/src/Makefile.am b/alberta_util/src/Makefile.am
index 30e54ae..3067634 100644
--- a/alberta_util/src/Makefile.am
+++ b/alberta_util/src/Makefile.am
@@ -2,6 +2,7 @@ AM_CPPFLAGS = -I$(builddir) -I$(srcdir) -I$(top_builddir) -I$(top_builddir)/incl
@OBSTACK_CPPFLAGS@

LIBNAME = alberta_utilities
+pkgconfigdir=$(libdir)/pkgconfig

lib_LTLIBRARIES = libalberta_utilities.la
pkgconfig_DATA = alberta-utilities.pc